keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -147,11 +151,13 @@ class TitusClusterHandler(
 
           val job = when {
             diff.isCapacityOnly() -> diff.resizeServerGroupJob()
+            diff.isEnabledOnly() -> diff.disableOtherServerGroupJob(resource, version.toString())
             else -> diff.upsertServerGroupJob(tagToUse) + resource.spec.deployWith.toOrcaJobProperties("Amazon")
           }
 
-          val description = when (version) {
-            null -> "Resize server group ${desired.moniker} in ${desired.location.account}/${desired.location.region}"
+          val description = when {
+            diff.isCapacityOnly() -> "Resize server group ${desired.moniker} in ${desired.location.account}/${desired.location.region}"
+            diff.isEnabledOnly() -> "Disable extra active server group ${job["asgName"]} in ${desired.location.account}/${desired.location.region}"
```

Comment:
Yep, it will. These other things are more of an optimization when there's something non-version related going on. On the AWS side there is a check for if it's capacity and scaling policies together. I don't know if it makes sense to do that optimization or not.